packages/dcos-integration-test/extra/test_dcos_diagnostics.py

```diff
@@ -407,6 +411,42 @@ def _get_bundle_list(dcos_api_session):
     return bundles
 
 
+@retrying.retry(wait_fixed=2000, stop_max_delay=120000,
+                retry_on_exception=lambda e: isinstance(e, NotCriticalException))
```

Comment:
The retry_on_exception could be `retry_on_exception=(NotCriticalException,))` - No need to check for the instance. The library does that for us.

Reply:
This is still not addressed.

Reply:
Rather than making a whole new exception for this singular purpose, can we just use `retry_on_result`? and then you `return False` in the code whenever you want to retry rather than raising this purpose-built exception